lib/resources.js

```diff
@@ -775,21 +790,18 @@ const buildElasticQuery = function (params) {
 
   if (params.q) {
     // Merge keyword-specific ES query into the query we're building:
-    query.bool = buildElasticQueryForKeywords(params).bool
+    query.bool = { must: [ buildElasticQueryForKeywords(params) ] }
   }
 
   const advancedParamQueries = ADVANCED_SEARCH_PARAMS
     .filter((param) => params[param])
     .map((param) =>
-      buildElasticQueryForKeywords({ q: params[param], search_scope: param }).bool.should
```

Comment:
buildElasticQueryForKeywords returned a `{bool: { should: ...}` because for a given search_scope, there are sometimes multiple clauses necessary (that need to be bound in a `should`.